docs/_sources/week02/spring-application/index.rst.txt

```diff
@@ -0,0 +1,239 @@
+:orphan:
+
+.. _week2_spring-application:
+
+=============================================================
+Project Week2 Setup: Spring, Git, GitLab, PostGIS, & IntelliJ
+=============================================================
+
+Before we can begin working on the requirements for this week, we will need to first setup the tools we will be using throughout this week.
+
+Tasks:
```

Comment:
I like presenting the steps in the body of the article in the hope of having students really think about what they need to do to solve the problem. It can also serve as a point for instructor led discussion. I will leave it.